code/Example/Drasil/DocumentLanguage/Definitions.hs

```diff
@@ -158,7 +158,7 @@ mkIMField i _ l@(InConstraints) fs  = (show l,
   foldr ((:) . eqUnR) [] (map tConToExpr (i ^. inCons))) : fs
 mkIMField i _ l@(OutConstraints) fs = (show l,
   foldr ((:) . eqUnR) [] (map tConToExpr (i ^. outCons))) : fs
-mkIMField i _ l@(Notes) fs = (show l, maybe [Paragraph EmptyS] (map Paragraph) (i ^. getNotes)) : fs
+mkIMField i _ l@(Notes) fs = (show l, maybe [Paragraph $ S "Not applicable."] (map Paragraph) (i ^. getNotes)) : fs
```

Comment:
This is an improvement, but @JacquesCarette was correct that not displaying empty notes is the best option.

Reply:
So I think the 'right' thing to do is to change `[Paragraph $ S "Not applicable."] ` to `[]` and have the printer detect this as an empty `Notes` field and print nothing (including no header).

Reply:
Done in commit 38a401f.